### PR TITLE
[apps] Added tsbpd URI query option

### PR DESCRIPTION
**Pull request by maxlovic**

### CODE REVIEW

File: apps/socketoptions.hpp [hunk 1]
```diff
@@ -216,6 +216,7 @@ const SocketOption srt_options [] {
     { "inputbw", 0, SRTO_INPUTBW, SocketOption::POST, SocketOption::INT64, nullptr},
     { "oheadbw", 0, SRTO_OHEADBW, SocketOption::POST, SocketOption::INT, nullptr},
     { "latency", 0, SRTO_LATENCY, SocketOption::PRE, SocketOption::INT, nullptr},
+    { "tsbpd", 0, SRTO_TSBPDMODE, SocketOption::PRE, SocketOption::BOOL, nullptr},
```
Comment (ethouris) [inline apps/socketoptions.hpp]: I know this is shorter and more handy, but the so far standard was that it's the lowercase name of the option without the `SRTO_` prefix...

File: apps/socketoptions.hpp [hunk 2]
```diff
@@ -216,6 +216,7 @@ const SocketOption srt_options [] {
...
     { "tsbpddelay", 0, SRTO_TSBPDDELAY, SocketOption::PRE, SocketOption::INT, nullptr},
```
Comment (ethouris) [inline apps/socketoptions.hpp]: As the option fixing happens - shouldn't this be removed, as an alias to `latency`?

File: apps/socketoptions.hpp (hunk 1 above)
Comment (maxlovic) [reply apps/socketoptions.hpp]: Do you mean `tsbpdmode` instead of `tsbpd`?

File: apps/socketoptions.hpp (hunk 2 above)
Comment (maxlovic) [reply apps/socketoptions.hpp]: I saw you removed in somewhere in your PRs. So I left it as is. Would you like me to remove it in this PR?

Comment (maxlovic) [reply apps/socketoptions.hpp]: In PR #622 you delete `tsbpddelay`.

**Pull request merged**